wiki/Medals/en.md

```diff
@@ -1,59 +1,295 @@
+| October 2017 Spotlight | Clear all the beatmaps bundled in any of the Spotlights October 2017 beatmap packs ([osu!](https://osu.ppy.sh/beatmaps/packs/1292)/[osu!taiko](https://osu.ppy.sh/beatmaps/packs/1293)/[osu!catch](https://osu.ppy.sh/beatmaps/packs/1294)/[osu!mania](https://osu.ppy.sh/beatmaps/packs/1295)). |
+| November 2017 Spotlight | Clear all the beatmaps bundled in any of the Spotlights November 2017 beatmap packs ([osu!](https://osu.ppy.sh/beatmaps/packs/1302)/[osu!taiko](https://osu.ppy.sh/beatmaps/packs/1303)/[osu!catch](https://osu.ppy.sh/beatmaps/packs/1304)/[osu!mania](https://osu.ppy.sh/beatmaps/packs/1305)). |
+| December 2017 Spotlight | Clear all the beatmaps bundled in any of the Spotlights December 2017 beatmap packs ([osu!](https://osu.ppy.sh/beatmaps/packs/1331)/[osu!taiko](https://osu.ppy.sh/beatmaps/packs/1332)/[osu!catch](https://osu.ppy.sh/beatmaps/packs/1333)/[osu!mania](https://osu.ppy.sh/beatmaps/packs/1334)). |
+| January 2018 Spotlight | Clear all the beatmaps bundled in any of the Spotlights January 2018 beatmap packs ([osu!](https://osu.ppy.sh/beatmaps/packs/1354)/[osu!taiko](https://osu.ppy.sh/beatmaps/packs/1355)/[osu!catch](https://osu.ppy.sh/beatmaps/packs/1356)/[osu!mania](https://osu.ppy.sh/beatmaps/packs/1357)). |
+| February 2018 Spotlight | Clear all the beatmaps bundled in any of the Spotlights February 2018 beatmap packs ([osu!](https://osu.ppy.sh/beatmaps/packs/1379)/[osu!taiko](https://osu.ppy.sh/beatmaps/packs/1380)/[osu!catch](https://osu.ppy.sh/beatmaps/packs/1381)/[osu!mania](https://osu.ppy.sh/beatmaps/packs/1382)). |
+| March 2018 Spotlight | Clear all the beatmaps bundled in any of the Spotlights January/February 2017 beatmap packs ([osu!](https://osu.ppy.sh/beatmaps/packs/1405)/[osu!taiko](https://osu.ppy.sh/beatmaps/packs/1406)/[osu!catch](https://osu.ppy.sh/beatmaps/packs/1407)/[osu!mania](https://osu.ppy.sh/beatmaps/packs/1408)). |
+| April 2018 Spotlight | Clear all the beatmaps bundled in any of the Spotlights January/February 2017 beatmap packs ([osu!](https://osu.ppy.sh/beatmaps/packs/1430)/[osu!taiko](https://osu.ppy.sh/beatmaps/packs/1431)/[osu!catch](https://osu.ppy.sh/beatmaps/packs/1432)/[osu!mania](https://osu.ppy.sh/beatmaps/packs/1433)). |
+
+## Seasonal Spotlights
+
+The **Seasonal Spotlights medals** are medals that are being awarded upon the completion of certain beatmap packs that are being released as a part of the [Seasonal Spotlights](https://osu.ppy.sh/home/news/2018-11-01-beatmap-spotlights-summer-2018). In order to unlock the medals in this category, a player needs to play through all the maps *in any of the four gamemode packs* that is being contested on the corresponding Seasonal Spotlights ranking chart. The usage of either [difficulty reduction mods](/wiki/Game_modifier#difficulty-reduction) or [difficulty increase mods](/wiki/Game_modifier#difficulty-increase) is permitted.
```

Comment:
```suggestion
*Seasonal Spotlights medals** are medals that are awarded upon the completion of certain beatmap packs that are being released as a part of the [Seasonal Spotlights](https://osu.ppy.sh/home/news/2018-11-01-beatmap-spotlights-summer-2018). In order to unlock the medals in this category, a player needs to play through all the maps in any of the four gamemode packs that are contested on the corresponding Seasonal Spotlights ranking chart; the usage of either [difficulty reduction mods](/wiki/Game_modifier#difficulty-reduction) or [difficulty increase mods](/wiki/Game_modifier#difficulty-increase) is permitted.
```

Reply:
think i'll keep the italic and the full stop here but changed the rest